### ISSUE TITLE
OpenID Connect replaces OpenID 2.0

### ISSUE THREAD

**Issue by parejkoj** [arabek/demovibes#2]
When connecting to scenemusic.net via my gmail account, I get redirected to this page:

https://support.google.com/accounts/answer/6206245?p=openid&rd=1

So I cannot access my favorites, etc. I don't know if this is a simple drop-in fix or not.

**Comment by parejkoj** [comment]
Note: if scenemusic logins are not managed by demovibes, I apologize in advance!

**Comment by arabek** [comment]
Yes, we are aware of the issue. For now please use your email (from the google account you have used to sign in) address to set/recover your password on this page:
https://www.scenemusic.net/accounts/password/reset/

**Comment by parejkoj** [comment]
Thanks for the response.

Hmm... Using the password reset page gives a red message: "The user account associated with this e-mail address cannot reset the password."

I never created a scenemusic.net account, just used the OpenID login button for Google.

**Comment by arabek** [comment]
Please send me an email to "arab (the swirly thing) scenemusic (the dot) net" from the account associated with your Google OpenId profile so that i can make a change in the db.
